### ISSUE TITLE
Adding languages sometimes creates them with zero strings (empty)

### ISSUE THREAD

**Issue by fnetX** [WeblateOrg/weblate#12747]
### Describe the issue

On Codeberg's Weblate instance at https://translate.codeberg.org, we are increasingly seeing reports of empty translations where there are no strings to translate. Deleting the language and recreating often works.

### I already tried

- [X] I've read and searched [the documentation](https://docs.weblate.org/).
- [X] I've searched for similar filed issues in this repository.

### Steps to reproduce the behavior

1. create a language for a component
2. wait until it is created
3. see that it has no strings, although the source translation has them
4. delete the empty language
5. recreate it
6. see that it worked (there are strings present to translate)

### Expected behavior

The language is created with strings immediately.

### Screenshots

![image](https://github.com/user-attachments/assets/fbb967da-a3ae-4793-a199-4dff0bda1876)

### How do you run Weblate?

Other

### Weblate versions

Weblate 5.7.2 with some codeberg patches probably

### Weblate deploy checks

```shell
~~~
# docker exec --user weblate weblate-docker_weblate_1 weblate check --deploy
SystemCheckError: System check identified some issues:

CRITICALS:
?: (weblate.E018) Could not download avatar: 404 Client Error: Not Found for url: https://codeberg.org/user/avatar/noreply@weblate.org/32
	HINT: https://docs.weblate.org/en/weblate-5.7.2/admin/optionals.html#avatars

INFOS:
?: (weblate.I021) Error collection is not set up, it is highly recommended for production use
	HINT: https://docs.weblate.org/en/weblate-5.7.2/admin/install.html#collecting-errors
?: (weblate.I028) Backups are not configured, it is highly recommended for production use
	HINT: https://docs.weblate.org/en/weblate-5.7.2/admin/backup.html

System check identified 3 issues (1 silenced).
~~~
```

### Additional context

Here is some logs I could gather, but I don't see a difference between working language additions and broken ones.

~~~
gunicorn stderr | [2024-10-12 16:05:49,957: INFO/681324] ournet/main: scheduling push
gunicorn stderr | [2024-10-12 16:05:49,958: INFO/681324] ournet/main/es: committed ['/app/data/vcs/ournet/main/ournet/es/root.11tydata.json'] as user <user@users.noreply.translate.codeberg.org>
gunicorn stderr | [2024-10-12 16:05:49,963: INFO/681324] ournet/main: scheduling update in background
celery-celery stderr | [2024-10-12 16:05:49,975: INFO/683752] ournet/main: this revision has been already parsed, skipping update
celery-celery stderr | [2024-10-12 16:05:49,975: INFO/ForkPoolWorker-13052] ournet/main: this revision has been already parsed, skipping update
nginx stdout | 10.0.3.1 - - [12/Oct/2024:16:05:49 +0000] "POST /new-lang/ournet/main/ HTTP/1.1" 302 5 "https://translate.codeberg.org/" "useragent"
celery-celery stderr | [2024-10-12 16:05:50,021: INFO/683725] ournet/main: updating repository
celery-celery stderr | [2024-10-12 16:05:50,021: INFO/ForkPoolWorker-13051] ournet/main: updating repository
celery-celery stderr | [2024-10-12 16:05:50,261: INFO/683725] ournet/main: update took 0.24 seconds
celery-celery stderr | [2024-10-12 16:05:50,261: INFO/ForkPoolWorker-13051] ournet/main: update took 0.24 seconds
celery-celery stderr | [2024-10-12 16:05:50,268: INFO/683725] ournet/main: repository up to date at 763aade88aba868f31e1515fb22c4001d22cb97d
celery-celery stderr | [2024-10-12 16:05:50,268: INFO/ForkPoolWorker-13051] ournet/main: repository up to date at 763aade88aba868f31e1515fb22c4001d22cb97d
nginx stdout | 10.0.3.1 - - [12/Oct/2024:16:05:50 +0000] "GET /progress/ournet/main/es/?info=1 HTTP/1.1" 200 5522 "https://translate.codeberg.org/" "useragent"
celery-celery stderr | [2024-10-12 16:05:50,302: INFO/683725] ournet/main: pushing to remote repo
celery-celery stderr | [2024-10-12 16:05:50,302: INFO/ForkPoolWorker-13051] ournet/main: pushing to remote repo
~~~

**Comment by fnetX** [comment]
I appreciate any hints on how to further diagnose the issue. CC @Bubu

It happened at least with the following repositories. I'll try to see if there is some common setup that could lead to this.

- https://translate.codeberg.org/projects/ournet/index/de/
- https://translate.codeberg.org/projects/ournet/main/es/ (not yet recreated)
- https://translate.codeberg.org/projects/zughy-friends-minetest/audio_lib/ru/ (and many more components and languages for this project)
- some more users confirmed this behaviour

**Comment by nijel** [comment]
When you click on these links showing zero, are there actually zero strings? There was a possibility that the stats would not be updated in some cases, this is already fixed, see #12486 and related issues.

Are there any add-ons installed on these components? b264d2f805abbc85efb8e53cf73e21fb06665f50 might be a fix in that case.
